### PR TITLE
Add a po file to be translated in the TRANSLATOR document

### PR DESCRIPTION
**Pull request by trebmuh**

### CODE REVIEW

File: TRANSLATORS [hunk 1]
```diff
@@ -20,6 +20,7 @@ There are few supplementation translations that can be found here (again,
 showing the case for Russian):
 
 libs/gtkmm2ext/po/ru.po
+gtk2_ardour/appdata/po/fr.po
```
Comment (x42) [inline TRANSLATORS]: ..again showing the case for Russian..  :)

Comment (trebmuh) [inline TRANSLATORS]: Reworded it.

**Comment by trebmuh** [comment]
I reworded it and I added a notes about slashes and pipes from the IRC discussion.

**Comment by trebmuh** [comment]
@x42 : would you mind merge this one, then I can start working from a clean git master please?

**Comment by pauldavisthefirst** [comment]
manually rebased and merged.

**Comment by trebmuh** [comment]
:+1:

**Pull request closed without merging**